This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Thu, 6 Jun 2019 12:30:00 +0200
Subject: [PATCH 1/1] Disable MAGICKCORE_LOCALE_SUPPORT on MinGW-64

Because msvcrt doesn't have _free_locale. See:
https://github.com/ImageMagick/ImageMagick6/issues/42

diff --git a/magick/locale.c b/magick/locale.c
index 1111111..2222222 100644
--- a/magick/locale.c
+++ b/magick/locale.c
@@ -62,7 +62,7 @@
 /*
   Define declarations.
 */
-#if defined(MAGICKCORE_HAVE_NEWLOCALE) || defined(MAGICKCORE_WINDOWS_SUPPORT)
+#if (defined(MAGICKCORE_HAVE_NEWLOCALE) || defined(MAGICKCORE_WINDOWS_SUPPORT)) && !defined(__MINGW32__)
 #  define MAGICKCORE_LOCALE_SUPPORT
 #endif
 #define LocaleFilename  "locale.xml"
